### PR TITLE
Update Russian translations for Data Layers

### PR DESCRIPTION
**Pull request by D0n-A**
Improved the context of Russian localization.

### CODE REVIEW

File: DataLayers/i18n/ru.json [hunk 1]
```diff
@@ -1,71 +1,71 @@
...
 
     // crops: harvest
-    "crop-harvest.name": "Посевы: Готовность к сбору",
-    "crop-harvest.ready": "Готовы",
-    "crop-harvest.not-ready": "Не готовы",
-    "crop-harvest.not-enough-time": "Не хватает времени",
+    "crop-harvest.name": "Состояние готовности к сбору культур",
+    "crop-harvest.ready": "Готово к сбору",
+    "crop-harvest.not-ready": "Не готово к сбору",
+    "crop-harvest.not-enough-time": "Необходимо немного больше времени для созревания",
```
Comment (Pathoschild) [inline DataLayers/i18n/ru.json:38]: This message is used when the crops don't have enough time to ripen (so they'll die before you can harvest them). Just to make sure, does the new text convey that too?

Comment (D0n-A) [reply DataLayers/i18n/ru.json:38]: @Pathoschild, I think so. "crop-harvest.name" - state of readiness to harvest (as a common name); " crop-harvest.ready" - the crop is ready to be harvested before it dies; " crop-harvest.not-ready" - the crop is NOT ready to be harvested before it dies; " crop-harvest.not-enough-time " - wait, the culture is still maturing. Here is what I put in the localization in a short form. I hope I got the context right)

Comment (Pathoschild) [reply DataLayers/i18n/ru.json:38]: Looks like you accidentally switched `not-ready` and `not-enough-time`. Here's how they're used:
* `ready`: you can harvest now.
* `not-ready`: crop is still growing, but you can harvest it soon.
* `not-enough-time`: crop will die before it's ready to harvest.

Comment (D0n-A) [reply DataLayers/i18n/ru.json:38]: @Pathoschild, Yes, that's right. Not quite correctly understood at first, now I suggest this:

"crop-harvest.name": "Состояние готовности урожая к сбору",
"crop-harvest.ready": "Культура готова к сбору урожая",
"crop-harvest.not-ready": "Культура не готова: необходимо немного больше времени для созревания",
"crop-harvest.not-enough-time": "Невозможно собрать культуру: вы не успели собрать урожай",

Decryption:
"crop-harvest.name": "The state of readiness of the crop for collection",
"crop-harvest.ready": "The crop is ready for harvest",
"crop-harvest.not-ready": "The crop is not ready: it needs a little more time to grow",
"crop-harvest.not-enough-time": "It is impossible to collect the crop: you did not have time to harvest",

Don't hit me for my English)))

Comment (Pathoschild) [reply DataLayers/i18n/ru.json:38]: Thanks! The text should be short though (ideally just a few words). Here's how that text looks in-game at full zoom:
>

Comment (D0n-A) [reply DataLayers/i18n/ru.json:38]: @Pathoschild,
Then:

"crop-harvest.not-ready": "Культура не готова: нужно больше времени для созревания",

Comment (Pathoschild) [reply DataLayers/i18n/ru.json:38]: Can we keep them less than 40 characters or so, so it fits in the box?

Something like this (just an example, might not be good Russian):
```
Состояние готовности урожая к сбору:
   Культура готова к сбору урожая
   Культура не готова
   вы не успели урожай
```

Comment (D0n-A) [reply DataLayers/i18n/ru.json:38]: @Pathoschild,
Then another option - the simplest and shortest:

"crop-harvest.name": "Состояние готовности урожая к сбору",
"crop-harvest.ready": "Культура готова к сбору урожая",
"crop-harvest.not-ready": "Культура ещё не готова к сбору урожая",
"crop-harvest.not-enough-time": "Вы не успели собрать урожай",

Comment (Pathoschild) [reply DataLayers/i18n/ru.json:38]: That's great, thanks! Will you add it to your pull request, or do you want me to make the change after accepting it?

Comment (D0n-A) [reply DataLayers/i18n/ru.json:38]: Come on you)

**Review by Pathoschild — APPROVED** [review]

**Comment by Pathoschild** [comment]
Thanks again! Merged into `develop` for the next Data Layers update.

**Pull request merged**